src/main/java/org/reaktivity/k3po/nukleus/ext/internal/behavior/layout/StreamsLayout.java

```diff
@@ -33,13 +35,15 @@
 {
     private final RingBuffer streamsBuffer;
     private final RingBuffer throttleBuffer;
+    private final FileChannel fileChannel;
 
     private StreamsLayout(
         RingBuffer streamsBuffer,
-        RingBuffer throttleBuffer)
+        RingBuffer throttleBuffer, FileChannel fileChannel)
```

Comment:
This should really be picked up by checkstyle, please move argument to next line.